README.md

```diff
@@ -1,75 +1,73 @@
+[Gitter **Chat**](https://gitter.im/WrenSecurity/Lobby)  
+[![CDDL-licensed](https://img.shields.io/badge/license-CDDL-blue.svg)](license)
+[![Build Status](https://semaphoreci.com/api/v1/wrensecurity/wrenam/branches/sustaining-13-5-x/badge.svg)](https://semaphoreci.com/wrensecurity/wrenam)
+[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/WrenSecurity/wrenam.svg)](http://isitmaintained.com/project/WrenSecurity/wrenam "Average time to resolve an issue")
+[![Percentage of issues still open](http://isitmaintained.com/badge/open/WrenSecurity/wrenam.svg)](http://isitmaintained.com/project/WrenSecurity/wrenam "Percentage of issues still open")
 
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/WrenSecurity/Lobby)
-
-wren:AM is a CDDL-licensed, community fork of ForgeRock's OpenAM product (which in turn was a fork of Sun's / Oracle's OpenSSO). Though our project originated with code that ForgeRock, Oracle and Sun previously released, we are not affiliated with them in any way. 
+Wren:AM is a CDDL-licensed, community fork of ForgeRock's OpenAM product. Though our project originated with code that ForgeRock previously released, we are not affiliated with ForgeRock in any way. 
 
 ForgeRock no longer releases any of the most recent versions of their software under an open-source license. ForgeRock’s “Community Edition” versions are much older versions then their commercial offerings. Join our community for the latest and greatest. 
 
-See [wrensecurity.org](http://wrensecurity.org/) for more information or join us on [Gitter](https://gitter.im/WrenSecurity/Lobby) and have a chat with us!
-
-## sustaining/13.5 branch
-
-This branch, the `sustaining/13.5` branch, is a stable branch intended primarily for bug and security fixes.
+See [wrensecurity.org](https://wrensecurity.org/) and [timeforafork.com](http://www.timeforafork.com/) for more information.
 
-## Building wren:AM 
-
-In order to build wren:AM from source you need the following software installed:
+## Preparing Your Build Environment
+In order to build Wren:AM from source you need the following software installed:
 
 * Java (OpenJDK) >= 1.7
 * Apache Maven >= 3.1.0
 
-Because the fork is still young and we are still working on rebranding all the group ID's are still set to `org.forgerock`. It is advised to clear out your Maven artifact cache (`~/.m2/repository`) before building. Failing to do so will likely cause problems with non-wren artifacts being used which in turn will cause build failures.
+## About This Branch (`sustaning/13.5.x`)
+The `sustaining/13.5.x` branch is a stable branch, intended for bug and security fixes against versions of Wren:AM that were originally part of ForgeRock's OpenAM 13.5.0 half-point release. The code in this release was provided to subscribers through [ForgeRock Backstage](https://backstage.forgerock.com) under the CDDL license.
```

Comment:
Since this is a bit delicate I think we attempt to be as clear as possible. What do you think about the following:

> The `sustaining/13.5.x` branch is a stable branch, intended for bug and security fixes. This branch originated from OpenAM's publicly available 13.5.0 RC11 sources. The branch was fast forwarded to 13.5.0 final (RC17) with CDDL licensed sources available to ForgeRock subscribers through [ForgeRock Backstage](https://backstage.forgerock.com). See commit 813d5a5e23e2fad5339fc2e33d161ca67fe5ecdc.